src/tls.rs

```diff
@@ -152,6 +179,47 @@ impl Context {
         })
     }
 
+    #[cfg(windows)]
+    fn load_ca_certs(&mut self) -> Result<()> {
```

Comment:
This method name is still pretty generic for being Windows-specific.

How about you define a separate implementation of the method for ``#[cfg(not(windows))]` that only does the `SSL_CTX_set_default_verify_paths()` call, so then in `new()` you'd just call `load_ca_certs()` unconditionally.

Reply:
I've done this but don't have a non-windows machine handy, so may find it breaks